File: src/client/mobile/app/src/main/java/com/superman/movieticket/domain/entities/Screening.kt
package com.superman.movieticket.domain.entities

import com.superman.movieticket.core.entity.BaseEntity
import java.io.Serializable

data class Screening (
    val movieId: String,
    val roomId: String,
    val startDate: String,
    val movie: Movie,
    val room: Room
) : BaseEntity(),Serializable

File: src/server/BuildingBlock/MovieTicket.Domain/Entities/Screening.cs
using MovieTicket.Core.Domain;

namespace MovieTicket.Domain.Entities;

public class Screening : EntityBase
{
    public Guid MovieId { get; set; }
    public Guid RoomId { get; set; }
    public DateTime StartDate { get; set; }
    public virtual Movie Movie { get; set; }
    public virtual Room Room { get; set; }
    public virtual ICollection<Reservation> Reservations { get; set; }
    
}